mesheryctl/internal/cli/root/config/config.go

```diff
@@ -53,15 +54,42 @@ func (mc *MesheryCtlConfig) CheckIfCurrentContextIsValid() (Context, error) {
 	if mc.CurrentContext == "" {
 		return Context{}, errors.New("current context not set")
 	}
+
 	if ctx, exists := mc.Contexts[mc.CurrentContext]; exists {
-		if ctx.Version == "" {
-			ctx.Version = "latest"
+		err := ctx.ValidateVersion()
+		if err != nil {
+			return Context{}, errors.New("invalid version " + ctx.Version + " specified")
```

Comment:
```suggestion
			return Context{}, err
```

Can't we just propagate back the error? @metonymic-smokey @aneeshnema

Reply:
@navendu-pottekkat isn't specifying with an error message clearer?

This is what I received when I tried this change:
![image](https://user-images.githubusercontent.com/48997495/112277921-441bd080-8ca8-11eb-9897-1c581f8aeeb6.png)

Reply:
Yes, specifying with an error message is clearer but we are already returning the same error message from the ValidateVersion function. So we don't need to print out the same error twice.

I don't think @navendu-pottekkat' s suggestion is causing the above error, can you please check again? (your validate version is accepting bob as a valid version)

Reply:
@aneeshnema 
If I just return `err`, it is always `<nil>` irrespective of whether the string matches the regex or not(the boolean `matched` however, reflects correctly). If I return a string using `errors.New`, I'm avoiding that bug.  

I ran the `ValidateVersion` function separately to observe it:
![image](https://user-images.githubusercontent.com/48997495/112289125-0624a980-8cb4-11eb-9991-3d099491d11d.png)

![image](https://user-images.githubusercontent.com/48997495/112289212-1c326a00-8cb4-11eb-9ec2-610f28d04418.png)

If we want to avoid double error messages, maybe I can change the return type to boolean returning `matched` and modify accordingly?

Reply:
@metonymic-smokey That should not be the case, if the version is not valid, it would return an error in line 59. Instead of again creating a new error, we can just pass back the error which is exactly the same. This would avoid making two instances of the same error.

Reply:
I checked this out and it works as expected. We can just pass the error instead of creating a new error.